### PR TITLE
Sj epic queue changes

### PR DESCRIPTION
**Pull request by Stuart-Johnson**

### CODE REVIEW

**Comment by kyle-glick** [comment]
@Stuart-Johnson Can you fix the merge conflicts on this branch?

File: db/migrate/20161109162634_add_origin_to_epic_queue_record.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddOriginToEpicQueueRecord < ActiveRecord::Migration
+  def change
+    add_column :epic_queue_records, :origin, :string
+    add_column :epic_queue_records, :identity_id, :integer
```
Comment (wtholt) [inline db/migrate/20161109162634_add_origin_to_epic_queue_record.rb]: Should this be a reference instead? (http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference)

File: db/migrate/20161115153644_add_identity_id_to_epic_queue.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddIdentityIdToEpicQueue < ActiveRecord::Migration
+  def change
+    add_column :epic_queues, :identity_id, :integer
```
Comment (wtholt) [inline db/migrate/20161115153644_add_identity_id_to_epic_queue.rb]: Should this be a reference instead? (http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference)

File: db/migrate/20161109162634_add_origin_to_epic_queue_record.rb (hunk 1 above)
Comment (Stuart-Johnson) [inline db/migrate/20161109162634_add_origin_to_epic_queue_record.rb]: Meh, as far as I can tell, the actual end result is identical in this instance, but I'll change it just for consistency.

**Pull request merged**